components/gc-most-requested/_base.scss

```
@@ -1,6 +1,6 @@
-/* Provisional most requested component */
+/* Most requested component */
 
-.provisional.gc-most-requested {
+.gc-most-requested {
 	border: none;
 
```

Remove the classes "well well-sm" from the HTML and extend those classes in the SCSS. That way it is easier to implement for authors.
```suggestion
@extend .well;
@extend .well-sm;
```

---

Done, I think. I couldn't accept the commit as it was out of date to the file.